### DISCUSSION TITLE
Clean up Issues

### DISCUSSION THREAD

**Discussion by ritwikvd** [redux-saga/redux-saga#2157] [General]
Yo! I'm seeing a ton of issues that aren't real "issues" per se. Questions and random queries appear quite commonplace after a cursory glance thru it.

Since this repo isn't really undergoing active development, I would highly recommend cutting down on all the bloat present in Issues so that this comes across as a stable lib, instead of an abandoned one. This would also help maintainers quickly ascertain pressing problems, and effectively prioritize. 

By default, you could perhaps create a system where discussions go here, in Discussions, and only after validation from a maintainer, does it become an issue.

I love this lib and nearly 200 issues with the last release in 2019 isn't a good look.

**Comment by Andarist** [comment]
I agree - managing issues takes time though and I don't have a lot of it. Would u like to help out by cleaning up such no real issues?

**Reply by neurosnap** [reply]
I would love to help clean up the GH issues and PRs.  Many of the PRs are dependabot.  Happy to help anyway I can
